### ISSUE TITLE
WinForms PrintPreviewControl ForeColor displays incorrectly only when set to White

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#14420]
### .NET version

11.0.100-preview.3.26175.115

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

It works on .NET 8.0, and it starts repro from .NET 9.0

1. After PR: https://github.com/dotnet/winforms/pull/13863 fixed in .NET 11, PrintPreviewControl ForeColor displays incorrectly only when set to White
2. Since this PR was not merged into .NET 9.0 and 10.0, all custom ForeColor settings on PrintPreviewControl are not rendered correctly

### Issue description

The ForeColor of PrintPreviewControl was set as **White** in properties window, however, it appears **Black** in the form designer.

.NET 11.0 project:
<img width="1367" height="403" alt="Image" src="https://github.com/user-attachments/assets/0eb135db-332b-40aa-828d-ab83b86a9d45" />

.NET 8.0 project:
<img width="1208" height="240" alt="Image" src="https://github.com/user-attachments/assets/a86953c1-cb41-413f-ad90-77a0108eabca" />

### Steps to reproduce

1. Create a WinForms .NET application
2. Add a PrintPreviewControl to form designer, observe that the ForeColor in Properties window is **White**, in form designer is **Black**
3. Build and run the application, it's same as in design time

**Comment by Olina-Zhang** [comment]
Tested in the latest .NET 11 Preview 7 build: 11.0.100-preview.7.26352.103 + built private dls from the main branch of Winforms repo, it was fixed for PrintPreviewControl's foreColor when setting its value to different color, includes White.

https://github.com/user-attachments/assets/d0caa5a9-f258-439f-a1e5-a9c06d697433
